web-client/src/presenter/actions/TrialSessionWorkingCopy/preparePrintableFormattedCasesAction.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * preparePrintableFormattedCasesAction
+ *
+ * @param {object} providers.props the cerebral props object
+ * @returns {Object} formattedCases
+ */
+
+export const preparePrintableFormattedCasesAction = ({ props }) => {
+  let { formattedCases } = props;
+  let temporaryFormattedCases = [];
+
+  formattedCases.map(formattedCase => {
+    temporaryFormattedCases.push(formattedCase);
+    if (formattedCase.leadCase) {
+      temporaryFormattedCases = temporaryFormattedCases.concat(
+        formattedCase.consolidatedCases,
+      );
+    }
+  });
+
+  formattedCases = temporaryFormattedCases;
+  return { formattedCases };
```

Comment:
would something like this make sense? maybe it's only more readable to me. trying to figure out a way to not use a temp variable but i guess that could be ok. the following assumes that we'd only expect `formattedCases` to contain one lead case. and you'd have to change the test's expectation to include `expect.arrayContaining`...food for thought i guess!

```suggestion
    let { formattedCases } = props;

  const leadCaseConsolidatedCases = formattedCases.filter(
    formattedCase => formattedCase.leadCase,
  )[0].consolidatedCases;

  leadCaseConsolidatedCases.forEach(consolidatedCase => {
    formattedCases.push(consolidatedCase);
  });

  return { formattedCases };
```

Reply:
@rosiehsmith The current implementation groups the member cases with the lead case, but if I am reading yours correctly, it would push member cases to the end of the formatted cases array which would be ungrouping the consolidated cases.

Reply:
ah i see, i didn't realize that was part of what was required for this action. rather than pushing to the end of `formattedCases` it could be a `splice` or something to be inserted after the leadCase. meh

Reply:
@rachelschneiderman, Rosie mentioned you might have an idea about this that would maybe let us not use the swap variable, but be simple to read. I hadn't used a splice because I felt like keeping track of an index for the splice is error-prone for the future and harder to keep track of in cognitive complexity terms.

Reply:
I was thinking this looked familiar to how `CaseListPetitioner` renders consolidated cases and when I looked into how that was done, it seems a lot of the sorting is done server-side and the client mostly just renders whatever it's handed back. I was just thinking it may be possible to DRY up consolidated case list logic a bit if we could reuse some of that code!